dynalab/tasks/task_io.py

```diff
@@ -0,0 +1,333 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import hashlib
+import json
+import os
+import uuid
+
+import requests
+from ts.context import Context
+
+from dynalab.tasks.annotation_mock_data import annotation_mock_data_generators
+from dynalab.tasks.annotation_verifiers import annotation_verifiers
+from dynalab_cli.utils import SetupConfigHandler
+
+
+class TaskIO:
+    def __init__(self, task_code, task_info_path=None):
+
+        if task_info_path is not None:
+            self.task_info = TaskIO.get_json_from_path(task_info_path)
+        else:
+            paths_to_check = [
+                f"./.dynalab/{task_code}.json",
+                f"/home/model-server/code/{task_code}.json",
```

Comment:
Better abstract `/home/model-server/code` into the `ROOTPATH` variable. You can define it here and import from here in base handler to avoid circular dependency. 

On coding practice here, you can do something like 
`paths_to_check = [task_info_path] if task_info_path else ["path1", "path2"]` 
and go to line 29 directly.

Reply:
I noticed that we don't use the ROOTPATH in base handler. It is used only in the handler template. In that case, do you think it's a good idea to import ROOTPATH from `task_io.py` file directly in the handler template and not to the base handler?

Reply:
What I meant was that, because handler will import `task_io`, defining `ROOTPATH` in `task_io` may cause circular dependency.

Reply:
Sorry just noticed my typo! 

I meant You can define it here and import from here THAN in base handler to avoid circular dependency.